src/components/task.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Item from './item';
+
+const Task = props => (
+  <li className="task__item">
+    {props.variety !== '' ?
```

Comment:
Yeah, the main one that occurred to me is 0 (falsy, but a meaningful number), but the way you're using it I'm not sure whether "variety 0" would make sense? But I also endorse not forcing your assumptions on the user... Still, explicitly setting null or undefined in the data seems like it shouldn't show up?